Manual.md

```diff
@@ -1101,7 +1101,13 @@ still be passed in if it's a GNU configure script.
 
 - `post_build()` Actions to execute after `do_build()`.
 
-- `pre_install()` Actions to execute after `post_build()`.
+- `pre_check()` Actions to execute after `post_build()`.
+
+- `do_check()` Actions to execute to run checks for the package.
+
+- `post_check()` Actions to execute after `do_check()`.
+
+- `pre_install()` Actions to execute after `post_build()` or `post_check()`.
```

Comment:
Check phase is no more optional than any other, please mention only one post_check here.

Reply:
I was under the impression from the manual's description of `make_check` that `*_check()` won't always run. Same impression from the complete lack of running checks when doing anything but `xbps-src check` in an unmodified void-packages checkout.

Reply:
That's true, yet no different from other phases. If template define no `do_configure` and no `post_configure`, the `pre_build` still do run.
The list is about relative order and purpose, not special cases like disabled tests or continuing build with `-f`.

Reply:
I see, will fix it